pkg/database/vercode.go

```diff
@@ -45,11 +45,13 @@ var (
 // VerificationCode represnts a verification code in the database.
 type VerificationCode struct {
 	gorm.Model
-	Code        string `gorm:"type:varchar(20);unique_index"`
-	Claimed     bool   `gorm:"default:false"`
-	TestType    string `gorm:"type:varchar(20)"`
-	SymptomDate *time.Time
-	ExpiresAt   time.Time
+	Code                   string `gorm:"type:varchar(20);unique_index"`
+	Claimed                bool   `gorm:"default:false"`
+	TestType               string `gorm:"type:varchar(20)"`
+	SymptomDate            *time.Time
+	ExpiresAt              time.Time
+	IssuingUserID          uint
```

Comment:
w/ Gorm - I think this can be 

```
IssuingUser *User
IssuingApp *AuthorizedApp
```

and I think that simplifies the rest of the code a bit as well.